prow/plugins/config.go

```diff
@@ -411,9 +411,16 @@ func (l Label) IsRestrictedLabelInAdditionalLables(restricted string) bool {
 }
 
 type RestrictedLabel struct {
-	Label        string   `json:"label"`
-	AllowedTeams []string `json:"allowed_teams,omitempty"`
-	AllowedUsers []string `json:"allowed_users,omitempty"`
+	Label        string          `json:"label"`
+	AllowedTeams []string        `json:"allowed_teams,omitempty"`
+	AllowedUsers []string        `json:"allowed_users,omitempty"`
+	AssignOn     []AssignOnLabel `json:"assign_on,omitempty"`
+}
+
+// AssignOnLabel specifies the label(s) that would trigger the RestrictedLabel.AllowedUsers'
```

Comment:
```suggestion
// AssignOnLabel specifies the label that would trigger the RestrictedLabel.AllowedUsers'
```

There can be multiple `AssignOnLabel` but each of them can only have one label